x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/persistence/DataFrameTransformsConfigManager.java

```diff
@@ -235,7 +240,7 @@ public void expandTransformIds(String transformIdsExpression, PageParams pagePar
                                     requiredMatches.unmatchedIdsString())));
                         return;
                     }
-                    foundIdsListener.onResponse(ids);
+                    foundIdsListener.onResponse(new Tuple<>(totalHits, ids));
```

Comment:
[optional] Alternatively you could use Tuple.tuple factory method instead of a constructor here.